swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, id=None, condition=0, age=100):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        self.age = age
+
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+    
+    def condition_description(self):
+        if self.condition == 5:
```

Comment:
for big 0 --  we should have stored the condition and descriptions in a dictionary. Right now it is having to go through and check is self.condition the same as 5, 4, 3, 2, 1, 0 (making it o(n)) but if it was in a dictionary it could just look it up and it would be o(1). 
If I am understanding this correctly.